### PR TITLE
WIP: Czech website

### LINKED ISSUE #773: Czech translation
Thanks to @skvrnami we are almost ready to prepare the website and deploy a new language! 🥳 

There are still a few things TODO, I'll probably miss some, but this issue can be used to track our progress.

- [x] Domain name
	- @skvrnami Do you have an idea for a nice domain name? It should be a translation of "datarequests" or something similar. 
- [ ] After the translation part is done, we have our own list for "code"-stuff at https://github.com/datenanfragen/website/blob/master/src/i18n/README.md
	- We'll do that, you don't have to. :D  
- [x] language maintaining
	- @skvrnami Would you be up to maintaining the translated website in the future? That means checking incoming translations from PRs or updating templates and strings if we make changes. Shouldn't be too much work, but it would be amazing, if we have a contact person for the translated languages we can not speak ourselves. :D

### PR DESCRIPTION
**Pull request by mal-tee**
Fixes #773 

## TODO
- [x] logo images
- [ ] i18n names
	- Seems to deploy without [every translation for the new language in `i18n-widget`](https://github.com/datenanfragen/website/blob/master/src/i18n/README.md?plain=1#L9), but it only shows an empty string. 
-  I am not sure if I even have the rights to do these:
	- [ ] [dattel foo](https://github.com/datenanfragen/website/blob/master/src/i18n/README.md?plain=1#L21-L22)
	- [ ] CDN? 
	- [x] DNS?

## Comment
- Do we need to set slugs for the required `.md` files in `content`?
- I had to commit this via `--no-verify`, since eslint complained about seemingly unrelated stuff. Probably because this PR touched these for the first time since we introduced new rules?

<details>
<summary>eslint error output</summary>

``` 
✖ eslint --ext .json --ext .js --ext .jsx --ext .ts --ext .tsx:

/home/mal/dade/website/components-package/src/index.ts
  5:37  error  Unable to resolve path to module './generated/globals.json'  import/no-unresolved
  7:25  error  Unable to resolve path to module '../package.json'           import/no-unresolved

/home/mal/dade/website/deploy-dattel.js
   1:19  error  Requires 'eslint-enable' directive for 'no-console'  eslint-comments/disable-enable-pair
  21:55  error  '_' is defined but never used                        @typescript-eslint/no-unused-vars
```

</details>

### CODE REVIEW

**Comment by baltpeter** [comment]
I have:

* Created the logo and added that in a commit.
* Set the correct DNS records and nameservers for zadostioudaje.org.
* Configured redirects for zadostioudaje.org -> www.zadostioudaje.org.
* Created the dattel and Bunny site.

**Comment by baltpeter** [comment]
> Do we need to set slugs for the required `.md` files in `content`?

No. In fact, I prefer to use the English slugs, but some translators wanted to translate them. But if you don't set a slug, you don't need aliases for the English slugs either.

**Comment by baltpeter** [comment]
> I had to commit this via `--no-verify`, since eslint complained about seemingly unrelated stuff. Probably because this PR touched these for the first time since we introduced new rules?

Um, I can't reproduce that. I only get a few warnings that we should fix but have been there for a while (we should really go the `-Werror` route, warnings don't work).

But I did fix the TypeScript errors that were the cause for this problem:

> Seems to deploy without [every translation for the new language in i18n-widget](https://github.com/datenanfragen/website/blob/master/src/i18n/README.md?plain=1&rgh-link-date=2023-03-29T20%3A12%3A11Z#L9), but it only shows an empty string.

You didn't provide a translation for `language-desc-cs`, that's what's shown in the `I18nWidget`.

**Comment by baltpeter** [comment]
Other than that, I didn't look at your commit, I'll leave that to @zner0L.

File: content/de/privacy.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Diese Datenschutzerklärung gilt für alle Aktivitäten des Datenanfragen.de e.&
 
 Dazu zählen zunächst die Daten, die wir von unseren Mitgliedern erheben und verarbeiten, aber auch jene die anfallen, wenn Du z.&nbsp;B. an uns spendest.
 
-Weiterhin gilt die Datenschutzerklärung für unsere Webseiten [Datenanfragen.de](https://www.datenanfragen.de), [datarequests.org](https://www.datarequests.org), [demandetesdonnees.fr](https://www.demandetesdonnees.fr), [solicituddedatos.es](https://www.solicituddedatos.es), [osobnipodaci.org](https://www.osobnipodaci.org), [gegevensaanvragen.nl](https://www.gegevensaanvragen.nl) und [pedidodedados.org](https://www.pedidodedados.org/).
+Weiterhin gilt die Datenschutzerklärung für unsere Webseiten [Datenanfragen.de](https://www.datenanfragen.de), [datarequests.org](https://www.datarequests.org), [demandetesdonnees.fr](https://www.demandetesdonnees.fr), [solicituddedatos.es](https://www.solicituddedatos.es), [osobnipodaci.org](https://www.osobnipodaci.org), [gegevensaanvragen.nl](https://www.gegevensaanvragen.nl) [pedidodedados.org](https://www.pedidodedados.org/) und [zadostioudaje.org/](https://zadostioudaje.org/).
```
Comment (baltpeter) [inline content/de/privacy.md:26]:
```suggestion
Weiterhin gilt die Datenschutzerklärung für unsere Webseiten [Datenanfragen.de](https://www.datenanfragen.de), [datarequests.org](https://www.datarequests.org), [demandetesdonnees.fr](https://www.demandetesdonnees.fr), [solicituddedatos.es](https://www.solicituddedatos.es), [osobnipodaci.org](https://www.osobnipodaci.org), [gegevensaanvragen.nl](https://www.gegevensaanvragen.nl) [pedidodedados.org](https://www.pedidodedados.org/) und [zadostioudaje.org](https://zadostioudaje.org/).
```

File: deploy-dattel.js [hunk 2]
```diff
@@ -12,13 +12,13 @@ if (!process.env.DATTEL_SERVER || !process.env.DATTEL_TOKEN) {
...
     try {
-        const languages = ['de', 'en', 'fr', 'pt', 'es', 'hr', 'nl'];
+        const languages = ['de', 'en', 'fr', 'pt', 'es', 'hr', 'nl', 'cs'];
 
         for (const language of languages) {
             const site_id = `dade-website-${language}`;
 
             // Cancel any stale deploys. This call will fail if there was no stale deploy, so we just ignore the error.
-            await dattel.cancelDeploy(site_id).catch((_) => {});
+            await dattel.cancelDeploy(site_id).catch();
```
Comment (baltpeter) [inline deploy-dattel.js:21]: An empty catch doesn't catch.

File: package.json [hunk 3]
```diff
@@ -42,7 +42,8 @@
         "prettier": "prettier --write {src,scripts,cypress,assets}/**/*.{js,jsx,ts,tsx,css,scss,json}",
         "deploy-dattel": "node deploy-dattel.js",
         "postinstall": "patch-package",
-        "build-components-package": "yarn tsm scripts/build-components-package.ts"
+        "build-components-package": "yarn tsm scripts/build-components-package.ts",
+        "build-sample-letters": "npx ./scripts/build-sample-letters/build.js"
```
Comment (baltpeter) [inline package.json:46]: Let's just use node.

File: scripts/build-sample-letters/res/i18n.json [hunk 4]
```diff
@@ -251,5 +255,71 @@
...
+        "access-body-010": "V případě, že ano, v souladu s čl. 15, odst. 3, prosím o poskytnutí kopie <italic>všech</italic> osobních údajů týkajících se mé osoby, které zpracováváte včetně jakýchkoliv pseudonymizovaných dat o mé osobě podle článku 4, odst. 5 GDPR. Dále žádám o přístup k následujícím informacím podle článku 15, odst. 1 GDPR:\n1. účely zpracování;\n2. kategorie dotčených osobních údajů;\n3. příjemci nebo kategorie příjemců, kterým osobní údaje byly nebo budou zpřístupněny, zejména příjemci ve třetích zemích nebo v mezinárodních organizacích;\n4. plánovaná doba, po kterou budou osobní údaje uloženy, nebo není-li ji možné určit, kritéria použitá ke stanovení této doby;\n5. veškeré dostupné informace o zdroji osobních údajů, pokud nejsou získány od subjektu údajů;\n6. skutečnost, že dochází k automatizovanému rozhodování, včetně profilování, uvedenému v čl. 22 odst. 1 a 4, a přinejmenším v těchto případech smysluplné informace týkající se použitého postupu, jakož i významu a předpoklá­daných důsledků takového zpracování pro subjekt údajů.",
+        "access-body-020": "",
```
Comment (baltpeter) [inline scripts/build-sample-letters/res/i18n.json:282]: Why are we not falling back?

**Review by mal-tee — COMMENTED** [review]
Thank you!

**Pull request merged**